documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,214 @@
+
+.Example curl request to add connector configuration for handling high volumes of messages
+[source,curl,subs=attributes+]
+----
+curl -X POST \
+  http://my-connect-cluster-connect-api:8083/connectors \
+  -H 'Content-Type: application/json' \
+  -d '{ "name": "my-source-connector",
+    "config":
+    {
+      "connector.class":"org.apache.kafka.connect.file.FileStreamSourceConnector",
+      "file": "/opt/kafka/LICENSE",
+      "topic":"my-topic",
+      "tasksMax": "4",
+      "type": "source"
+      "producer.override.batch.size": 327680
+      "producer.override.linger.ms": 100
+      "consumer.fetch.max.bytes": 52428800
+      "consumer.max.partition.fetch.bytes": 1048576
+      "consumer.max.poll.records": 500
+    }
+}'
+----
+
+== Configuring MirrorMaker 2.0 for high-volume messages 
+MirrorMaker 2.0 fetches data from the source cluster and hands it to the Kafka Connect runtime producers so that it's replicated to the target cluster.
+
+The following example shows the configuration for MirrorMaker 2.0 using the `KafkaMirrorMaker2` custom resource. 
+
+.Example MirrorMaker 2.0 configuration for handling high volumes of messages
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaMirrorMaker2ApiVersion}
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker2
+spec:
+  version: {DefaultKafkaVersion}
+  replicas: 1
+  connectCluster: "my-cluster-target"
+  clusters:
+  - alias: "my-cluster-source"
+    bootstrapServers: my-cluster-source-kafka-bootstrap:9092
+  - alias: "my-cluster-target"
+    config:
+      offset.flush.timeout.ms: 10000
+    bootstrapServers: my-cluster-target-kafka-bootstrap:9092
+  mirrors:
+  - sourceCluster: "my-cluster-source"
+    targetCluster: "my-cluster-target"
+    sourceConnector:
+      tasksMax: 2
+      config:
+        producer.override.batch.size: 327680
+        producer.override.linger.ms: 100
+        consumer.fetch.max.bytes: 52428800
+        consumer.max.partition.fetch.bytes: 1048576
+        consumer.max.poll.records: 500
+    # ...
+  resources: 
+    requests:
+      cpu: "1"
+      memory: Gi
+    limits:
+      cpu: "2"
+      memory: 4Gi      
+----
+
+== Checking the MirrorMaker 2.0 message flow
+
+If you are using Prometheus and Grafana to monitor your deployment, you can check the MirrorMaker 2.0 message flow.
+
+The example MirrorMaker 2.0 Grafana dashboards provided with Strimzi show the following metrics related to the flush pipeline.
+
+* The number of messages in Kafka Connect's outstanding messages queue
+* The available bytes of the producer buffer
+* The offset commit timeout in milliseconds
+
+You can use these metrics to gauge whether or not you need to tune your configuration based on the volume of messages.
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:{BookURLDeploying}#assembly-metrics-setup-{context}[Grafana dashboards^]
+* link:{BookURLDeploying}#assembly-creating-managing-connectors-{context}[Creating and managing connectors^]
```

Comment:
It looks like this page is focused on source connectors. If so this should be mentioned at the top

Reply:
I've added a few lines about sink connectors, as indicated earlier